xbmc/windowing/X11/GLContextGLX.h

```diff
@@ -35,6 +35,8 @@ class CGLContextGLX : public CGLContext
   virtual bool SwapBuffers(const CDirtyRegionList& dirty, int &mode);
   virtual void QueryExtensions();
   virtual bool IsExtSupported(const char* extension);
+  GLXWindow m_glxWindow;
+  GLXContext m_glxContext;
```

Comment:
the egl ones are exposed as public too http://www.amazon.com/review/R1R2N8UO4H40KM/ref=cm_cr_dp_title?ie=UTF8&ASIN=B0073HI94M&channel=detail-glance&nodeID=541966&store=pc
WinSystemX11GLContext wants to access them